### PR TITLE
Document report

### PR DESCRIPTION
**Pull request by Morvin-Ian**

### CODE REVIEW

**Review by coderboy-exe — COMMENTED** [review]
Looks good but I'm not sure why there is a url in line 27 of thr requirements file. Care to explain or look into that? Is it necessary or can it be removed? @Morvin-Ian

**Review by coderboy-exe — CHANGES_REQUESTED** [review]
Please look at my previous comment:

https://github.com/LL04-Finance-Dowell/100018-dowellWorkflowAi-testing/pull/182#pullrequestreview-1749708990

**Comment by coderboy-exe** [comment]
> > Yeah Its neccessary, The spacy model package can be either small, medium, or large. That 27th line downloads the smaller one which we are using.

All clear for merge @WawinyEdwin

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]

File: app/urls_v2.py [hunk 1]
```diff
@@ -64,4 +64,7 @@
         "notifications/<str:process_id>/reminders/",
         views_v2.NotificationReminder.as_view(),
     ),
+
+    path("reports/document/<str:item_id>/", views_v2.DocumentReport.as_view(), name="document-report"),
```
[the remark this answers is not in the document]
Comment (Morvin-Ian) [reply app/urls_v2.py:68]: Okay

**Review by WawinyEdwin — APPROVED** [review]

**Pull request merged**